tests/test_vxlan_global.rb

```diff
@@ -90,16 +88,14 @@ def test_dup_host_mac_detection_default
   def test_anycast_gateway_mac_set
     vxlan_global = VxlanGlobal.new
     mac_addr = '1223.3445.5668'
-    vxlan_global.enable
-    vxlan_global.anycast_gateway_mac_set('True', mac_addr)
+    vxlan_global.anycast_gateway_mac = mac_addr
     assert_equal(mac_addr, vxlan_global.anycast_gateway_mac,
                  'Error: anycast-gateway-mac mismatch')
   end
 
   def test_anycast_gateway_mac_clear
     vxlan_global = VxlanGlobal.new
-    vxlan_global.enable
-    vxlan_global.anycast_gateway_mac_set('False', '')
+    vxlan_global.anycast_gateway_mac = ''
     assert_equal('', vxlan_global.anycast_gateway_mac,
                  'Error: anycast-gateway-mac mismatch')
```

Comment:
You don't have a test for your default api. You can just change this test a little to remedy that:

```
def test_anycast_gateway_mac_clear
     vxlan_global = VxlanGlobal.new
    vxlan_global.anycast_gateway_mac = vxlan_global.default_anycast_gateway_mac
     assert_equal(vxlan_global.default_anycast_gateway_mac, vxlan_global.anycast_gateway_mac,
                  'Error: anycast-gateway-mac mismatch')
```